hack/generate-manifest.sh

```diff
@@ -432,6 +438,16 @@ if $HW_OFFLOAD; then
     cd ..
 fi
 
+if $SRIOV; then
+    mkdir sriov && cd sriov
+    cp ../../patches/sriov/sriov.yml .
```

Comment:
Maybe you forgot to include sriov.yml in this PR?

Reply:
Yes; added it now, thanks!